src/Symfony/Bridge/Twig/Extension/FormExtension.php

```diff
@@ -54,6 +56,8 @@ public function initRuntime(Environment $environment)
     {
         if ($this->renderer instanceof TwigRendererInterface) {
             $this->renderer->setEnvironment($environment);
+        } elseif ($this->renderer instanceof FormRendererInterface && $this->renderer->getEngine() instanceof TwigRendererEngineInterface) {
+            $this->renderer->getEngine()->setEnvironment($environment);
```

Comment:
you should do nothing in this case instead (it already has the environment injected properly)

Reply:
Ah ok. Done :wink: